api/src/pcapi/core/mails/transactional/finance_incidents/finance_incident_notification.py

```diff
@@ -41,6 +53,57 @@ def send_finance_incident_emails(finance_incident: finance_models.FinanceInciden
         )
 
         mails.send(
-            recipients=[venue.current_reimbursement_point.bookingEmail],
+            recipients=[booking_email],
             data=data,
         )
+
+
+def send_commercial_gesture_email(finance_incident: finance_models.FinanceIncident) -> None:
+    venue = finance_incident.venue
+
+    if not FeatureToggle.WIP_ENABLE_NEW_BANK_DETAILS_JOURNEY.is_active():
+        if not venue.current_reimbursement_point or not venue.current_reimbursement_point.bookingEmail:
+            return
+        booking_email = venue.current_reimbursement_point.bookingEmail
+    else:
+        if (
+            not venue.current_bank_account_link
+            or not venue.current_bank_account_link.bankAccount
+            or not venue.bookingEmail
+        ):
+            return
+        booking_email = venue.bookingEmail
+
+    offers_incidents: dict[tuple, list[finance_models.BookingFinanceIncident]] = {}
+    for booking_finance_incident in finance_incident.booking_finance_incidents:
+        booking = booking_finance_incident.booking or booking_finance_incident.collectiveBooking
+        if isinstance(booking, bookings_models.Booking):
+            offer = booking.stock.offer
+        else:
+            offer = booking.collectiveStock.collectiveOffer
+
+        if (offer.id, offer.name) not in offers_incidents:
+            offers_incidents[(offer.id, offer.name)] = []
+        offers_incidents[(offer.id, offer.name)].append(booking_finance_incident)
```

Comment:
[détail / style] En définissant `offers_incidents` comme un `defaultdict(list)` (plutôt qu'un simple dictionnaire), on peut simplifier ici : 

```suggestion
        offers_incidents[(offer.id, offer.name)].append(booking_finance_incident)
```